airbyte-webapp/src/core/resources/DbMigration.ts

```diff
@@ -0,0 +1,62 @@
+import BaseResource from "./BaseResource";
+import { MutateShape, ReadShape, Resource, SchemaDetail } from "rest-hooks";
+
+export interface DbMigrationInfoItem {
+  migrationType: string;
+  migrationVersion: string;
+  migrationDescription: string;
+  migrationState?: string;
+  migratedBy?: string;
+  migratedAt?: bigint;
+  migrationScript?: string;
+}
+
+export interface DbMigrationInfoResult {
+  migrations: Array<DbMigrationInfoItem>;
+}
+
+export interface DbMigrationMigrateResult {
```

Comment:
could you please move interface to `domain` directory? The idea is that resources directory is actually store ( or API ) layer and is responsible to receive data. But models themselves could be declared in some more common place.